src/APILookupPango.txt

```diff
@@ -148,10 +148,6 @@ code: end
 struct: Layout
 array: set_text text length
 
-struct: Map
-out: get_engines exact_engines
-out: get_engines fallback_engines
-
```

Comment:
Some of these might need to be put into a version block to still be able to generate with an older version of Pango

Reply:
Yes, you are right. I create a new commit with fix